kubernetes/opentelemetry-demo.yaml

```diff
@@ -1013,7 +1013,7 @@ data:
         {
           "datasource": {
             "type": "grafana-opensearch-datasource",
-            "uid": "P9744FCCEAAFBD98F"
+            "uid": "webstore-logs"
```

Comment:
The Kubernetes manifests file is generated. Please revert changes to this file for the PR.